zerver/lib/bulk_create.py

```diff
@@ -16,6 +18,7 @@ def bulk_create_realms(realm_list):
     Realm.objects.bulk_create(realms_to_create)
 
 def bulk_create_users(realms, users_raw, bot_type=None):
+    # type: (Dict[unicode, Realm], Set[Tuple[Any, Any, unicode, bool]], Optional[int]) -> None
```

Comment:
I replaced the first two `Any` arguments with `text_type` but got this result:

```
zerver/lib/bulk_create.py:53: error: Incompatible types in assignment (expression has type "unicode", variable has type "bool")
zerver/lib/bulk_create.py:63: error: Incompatible types in assignment (expression has type "unicode", variable has type "bool")
```

It looks like the first two arguments are sometimes `unicode` and sometimes `bool`. Is there anything other than `Any` we could use in this case?

Also: should I replace the two `unicode` arguments with `text_type`?